web/src/topic_link_util.ts

```diff
@@ -18,6 +18,8 @@ function get_stream_name_from_topic_link_syntax(syntax: string): string {
 
 export function escape_invalid_stream_topic_characters(text: string): string {
     switch (text) {
+        case "&":
+            return "&amp;";
         case "`":
             return "&grave;";
```

Comment:
Let’s also change this to `&#96;`, because `&grave;` is an HTML5 entity that should be totally fine in theory but [fails to be parsed by LXML](https://chat.zulip.org/#narrow/channel/9-issues/topic/HTML.20entities.20in.20email.20notifications/near/1937173) in practice.

Reply:
Done